packages/algob/src/internal/core/config/config-validation.ts

```diff
@@ -156,9 +159,7 @@ export function getValidationErrors(config: any): CfgErrors {
 				errors.push(net, "host", host, "hostname string (eg: http://example.com)");
 			}
 			const token = hcfg.token;
-			if (typeof token === "string" && token.length < 10) {
-				errors.push(net, "token", token, "string (with length > 10)");
-			} else if (typeof token !== "object" && typeof token !== "string") {
+			if (typeof token !== "object" && typeof token !== "string") {
```

Comment:
Could you check against `null` as well? Or could it only be `null`

Reply:
Yes, `Null` is also allow as well